spec/PeriodicBackgroundSync-index.bs

```diff
@@ -0,0 +1,75 @@
+<pre class='metadata'>
+Title: Web Periodic Background Synchronization
+Status: CG-DRAFT
+ED: https://wicg.github.io/BackgroundSync/spec/PeriodicBackgroundSync-index.html
+Shortname: periodic-background-sync
+Level: 1
+Editor: Mugdha Lakhani, Google, nator@chromium.org
+Abstract: This specification describes a method that enables web applications to periodically synchronize data and content in the background.
+Group: wicg
+Repository: WICG/BackgroundSync
+</pre>
+
+<pre class=link-defaults>
+spec:html; type:dfn; for:/; text:browsing context
+spec:service-workers; type:dfn; text:frame type
+</pre>
+
+Introduction {#intro}
+=====================
+
+  <em>This section is non-normative.</em>
+
+  Web Applications often run in environments with unreliable networks (e.g., mobile phones) and unknown lifetimes (the browser might be killed or the user might navigate away).
+  This makes it difficult for web apps to keep their content and state in sync with servers. This API is intended to reduce the time between content creation and content synchronization between the servers and the web app.
+
+  As this API relies on service workers, functionality provided by this API is only available in a [=secure context=].
+
+  ## Example ## {#example}
+  Requesting a [=periodic Background Sync opportunity=] at a mininimum interval of one day from a [=browsing context=]:
+
+  <pre class="lang-js">
+    async function registerPeriodicNewsCheck() {
+      const registration = await navigator.serviceWorker.ready;
+      try {
+        await registration.periodicSync.register('fetch-news', {
+          minInterval: 24 * 60 * 60 * 1000,
+        });
+      } catch (error) {
+        console.log('Periodic Sync could not be registered!');
+      }
+    }
+  </pre>
+
+  Reacting to a <code>periodicsync</code> event within a [=service worker=]:
+
+  <pre class="lang-js">
+    self.addEventListener('periodicsync', event => {
+      event.waitUntil(fetchAndCacheLatestNews());
+    });
+  </pre>
+
+  In the above example <code>fetchAndCacheLatestNews</code> is a developer-defined function.
+
+Concepts {#concepts}
+========================
+
+The <code>periodicsync</code> event is considered to run <dfn>in the background</dfn> if no [=service worker clients=] whose [=frame type=] is top-level or auxiliary exist for the origin of the corresponding service worker registration.
+
+The user agent is considered to be <dfn>online</dfn> if the user agent has established a network connection. A user agent MAY use a stricter definition of being [=online=]. Such a stricter definition MAY take into account the particular [=service worker=] or origin a [=periodicsync registration=] is associated with.
+
+A <dfn>periodic Background Sync opportunity</dfn> allows periodic synchronization between the server and the web app, the exact interval of which is decided by the user agent. It can be requested via a [=periodicsync registration=].
```

Comment:
Leave a TODO here, because the `periodicsync registration` is a tuple that contains data, whereas I think you'll want to refer to `PeriodicSyncManager.register()` instead.

Reply:
done